docs/faq.md

```diff
@@ -28,3 +28,9 @@
     - No, it is what it is. GADS uses Appium under the hood for the interactions so we are as fast as it allows us to be.
 - *I can load the device but interaction does not work at all/session expired popup appears, why?*
     - There is probably an issue with Appium setup or dependencies. It is quite possible to start Appium server successfully but everything fails due to missing environment variable like `ANDROID_HOME` or something in that line. Observe the respective device Appium logs either in UI or file
+- When I want to unlock my phone a session expired popup appears, why?
+  - Make sure your phone is not passcode protected since appium is only unlocking the device and it fails if it lands on the passcode screen 
```

Comment:
```suggestion
  - Make sure your phone is not passcode protected since Appium is only unlocking the device and it fails if it lands on the passcode screen 
```